app/src/main/java/org/wikipedia/analytics/eventplatform/AppsEvent.kt

```diff
@@ -0,0 +1,15 @@
+package org.wikipedia.analytics.eventplatform
+
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+import kotlinx.serialization.Transient
+
+@Serializable
+sealed class AppsEvent(@Transient val streamName: String = "") : Event(streamName) {
```

Comment:
Would it be possible to make it as `AppEvent` instead of a plural name of it?

Reply:
So these are the events that only the `apps` send Cooltey... then there is a provision for `legacy` events... just maintaining the same naming as the MEP repository... actually they name it `mobile_apps` so you feel it is better to name `MobileAppsEvent` on our side too?

Reply:
`MobileAppsEvent` sounds good to me.